### ISSUE TITLE
[feat] Provide an option to disable Measured Templates inclusion for Automatic Camera

### ISSUE THREAD

**Issue by 117ksk** [sPOiDar/fvtt-module-stream-view#78]
**Describe the bug**
When a actor has an active Aura effect on it and moves on the canvas, the automatic camera mode will zoom out to the Aura size and pan slightly downward.

**To Reproduce**
1. Set camera mode to automatic
2. Begin encounter and have camera track active combatant
3. Apply large range Aura effect to active combatant (60ft Aura works well)
4. Move selected combatant to new square
5. Watch stream view zoom all the way out and no longer focus selected combatant in center of view

**Expected behavior**
When combatant moves, camera should follow and be centered on actor. Ideally camera would not zoom out to view entire aura radius especially for a large radius aura.

**Environment (please complete the following information)**
 - FoundryVTT Version: v11.315
 - Stream View Version:  v2.0.3
 - Browser: Chrome

**Additional context**
A settings toggle to not include Auras in automatic camera would be nice.

**Comment by sPOiDar** [comment]
Is that a core foundry feature? I don't recall auras being a thing.

**Comment by 117ksk** [comment]
> Is that a core foundry feature? I don't recall auras being a thing.

Oh I’m sorry, must be a PF2e system thing not core. In either case, it looks like an aura simply places a circle template centered on an actor and moves with said actor.

**Comment by sPOiDar** [comment]
Ah right, I assume this only happens during combat?

Measured templates for the current combatant were included so you can see where they're casting spells. I guess I could add an option to disable this, but I doubt there's a sensible generalized way to omit only "aura" templates.

**Comment by 117ksk** [comment]
Yep, only when in combat encounters. I understand and typically appreciate the inclusion of templates due to spells and effects that should be captured by the automatic camera. A setting toggle would be nice however to choose to include this feature in the automatic camera mode. Or maybe even a way to include templates that are less than a specified size and exclude those that are too big?

**Comment by 117ksk** [comment]
I believe a simple toggle in settings to not include templates would work well for PF2e. Due to templates auto-targeting with the PF2e Toolbelt module, I believe the targets of the spells that normal templates effect would still be included. This would just prevent aura templates that do not auto target actors from zooming out when not wanted.
